src/main/java/io/vertx/codegen/CodeGen.java

```diff
@@ -83,9 +83,11 @@ public void init(RoundEnvironment round, ClassLoader loader) {
       TypeMirror type = null;
       if (converter.className != null) {
         TypeElement typeElt = elementUtils.getTypeElement(converter.className);
-        if (typeElt != null) {
-          type = typeElt.asType();
-        }
+		if (typeElt != null) {
```

Comment:
@vietj, this seems to be easier to test. We just need to write a class in the json-mapper.property wrongly.